paladin-quests.lic

```diff
@@ -0,0 +1,105 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#cleric-quests
+=end
+
+custom_require.call(%w[common common-items common-money common-travel drinfomon equipmanager])
+
+class PaladinQuests
+  include DRC
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    unless DRStats.paladin?
+      echo '***MUST BE A PALADIN***'
+      exit
+    end
+
+  arg_definitions = [
+      [
+        { name: 'warding', regex: /warding/i, description: 'Complete the Glyph of Warding Quest.' }
+      ],
+      [
+        { name: 'bonding', regex: /bonding/i, description: "Complete the Glyph of Bonding Quest." }
+      ],
+      [
+        { name: 'light', regex: /light/i, description: 'Complete the Glyph of Light Quest.' }
+      ],
+      [
+        { name: 'mana', regex: /mana/i, description: 'Complete the Glyph of Mana Quest.' }
+      ],
+      [
+        { name: 'ease', regex: /ease/i, description: 'Complete the Glyph of Ease Quest.' }
+      ],
+      [
+        { name: 'renewal', regex: /renewal/i, description: 'Complete the Glyph of Renewal Quest.' }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    pause
+
+    EquipmentManager.new.empty_hands
+
+    if args.warding
+      warding
+    elsif args.bonding
+      tamsine
+    elsif args.light
+      eluned
+    elsif args.mana
+      mana
+    elsif args.ease
+      ease
+    elsif args.renewal
+      renewal
+    else
+      echo '***UNKNOWN QUEST NAME***'
+    end
+  end
+
+  def warding
+    walk_to(8228)
```

Comment:
Does the guildleader give a visa for f2p paladins?

Reply:
yeah for Glyph of Warding, Glyph of Light, and Glyph of Mana

Reply:
He does, they'll need it before doing this. I won't be holding hands *that* much. :D